_gsocblogs/2022/blog_ROOT_Neelshah.md

```diff
@@ -0,0 +1,178 @@
+---
+project: ROOT
+title: Inference of Deep Learning models in TMVA/SOFIE
+author: Neel Shah
+photo: blog_authors/NeelShah.jpg 
+date: 24.07.2022 
+year: 2022 
+layout: blog_post 
+logo: ROOT-logo.png 
+intro: |
+Currently, we are developing a fast inference system in TMVA, called SOFIE, that takes takes ONNX model as input and produces compilation-ready standalone C++ scripts as output. These scripts will then provide users an easy way to deploy their deep learning models in their physics software and analysis frameworks. This project will focus on development of some missing deep learning operations which will allow to build more complex networks within TMVA. Specifically, we propose to implement the inference functionality of some ONNX operators in the code generation format. 
+---
+
+# Mid Evaluation Blog for GSOC 2022
+
+## Pre-GSOC Period
+Getting into GSOC was my dream since college years, On May 20, i was so delighted when i got into my preferred Project into one of the most competitive Organisations.I will forever remember those sleepless nights spent in solving those tasks, debugging, building the project !!
```

Comment:
```suggestion
Getting into GSOC was my dream since my college years, On May 20, I was so delighted when I got into my preferred Project in one of the most competitive Organisations. I will forever remember those sleepless nights spent solving those tasks, debugging, and building the project !!
```